### PR TITLE
Add Dependent functional test

### PR DESCRIPTION
**Pull request by mk08sh**
# Fixes or Implements [AB#9251](https://qslvic.visualstudio.com/304a1f8c-dace-4f85-adf3-bf563d5b3a39/_workitems/edit/9251)

* [x] Enhancement
* [ ] Bug
* [ ] Documentation

## Description

Added functional tests for the following:
-add dependent
-validate qualified/disqualified age
-validate cancel button
-validate register button 
There are some test id's missing:
-X icon is missing test id 
-error messages for each input is missing test id
-attestation checkbox is not visible to cypress which leads to using force:true 
## Testing

* [ ] Unit Tests Updated
* [x] Functional Tests Updated
* [ ] Not Required

### Steps to Reproduce

N/A

### UI Changes
NO

### Browsers Tested

* [x] Chrome
* [ ] Safari
* [ ] Edge
* [ ] Firefox

Provide an explanation for any test(s) not completed.

## Notes/Additional Comments

Any additional notes or comments that may be relevant.  Include any specialized deployment steps here.  

</br>
</br>
</br>
___
  
## Reviewer Guidelines

A reviewer should examine the PR if the Status above indicates the PR is **Ready**.
Once ready, the reviewer should read the information below as well as supporting information like the Azure ticket to ensure they have a good understanding of the request.

The reviewer should then:

1. Is the PR template complete and accurate?

    Confirm by reading all sections below.

1. Confirm the merge branch

    A normal feature/bug branch should merge into the Dev branch.  Exceptions for hotfix and sprint merges do occur and the target would then be Master.

1. Do the code commits have an appropriate comment and include linkage to the azure ticket by including a reference to "AB#nnnn"?

1. Review the Code Changes

    1. Do the changes match the description?

        The code commits should reflect the documented description and nothing more.  If additional changes are included the reviewer should call these out and request that the developer document the changes in Azure and in the description or remove the changed code.

    1. useful tips

        * Any obvious errors or omissions?
        * Does the code match the style of other similar work?
        * Is the code easy to understand?
        * Is the code well documented?
        * DB Migrations are appropriate?

If the Answer to any of the questions above is No, then a request for change should be the result of the review.

### CODE REVIEW

File: Apps/WebClient/test/functional/e2e/cypress/integration/dependent/dependents.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    it('Validate Disqualidfied Birthdate and Cancel Button', () => {
+        cy.get('[data-testid=addNewDependentBtn]')
+        .click();
+        cy.get('[data-testid=firstNameInput]')
+        .type('John');
+        cy.get('[data-testid=lastNameInput]')
+        .type('Tester');
+        cy.get('[data-testid=dateOfBirthInput]')
+        .type('1988-01-21');
+        cy.get('[data-testid=phnInput]')
```
Comment (sslaws) [inline Apps/WebClient/test/functional/e2e/cypress/integration/dependent/dependents.js:30]: You need to assert what happens after you type 1988...

File: Apps/WebClient/test/functional/e2e/cypress/integration/dependent/dependents.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+            .type('9874307215');
```
Comment (sslaws) [inline Apps/WebClient/test/functional/e2e/cypress/integration/dependent/dependents.js:31]: And you should assert what happens when you clear and then when you type again.

File: Apps/WebClient/test/functional/e2e/cypress/integration/dependent/dependents.js [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+        cy.get('[data-testid=dateOfBirthInput]')
+        .clear()
+        .type('2005-01-01');
+    
+        cy.get('[data-testid=cancelRegistrationBtn]')
+            .click();
+    });
+
+    it('Validate Add', () => {
```
Comment (sslaws) [inline Apps/WebClient/test/functional/e2e/cypress/integration/dependent/dependents.js:40]: ditto:  needs assertions

**Review by sslaws — CHANGES_REQUESTED** [review]

**Comment by leobcgov** [comment]
We have copied these tests to the https://github.com/bcgov/healthgateway/pull/1222 PR.

**Pull request closed without merging**